Doc/tools/extensions/pyspecific.py

```diff
@@ -73,9 +73,11 @@ def issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
 
 def gh_issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     issue = utils.unescape(text)
-    # sanity check: all GitHub issues have ID >= 32426
+    # sanity check: new GitHub issues have ID >= 28525
     # even though some of them are also valid BPO IDs
-    if int(issue) < 32426:
+    # Note: there are actually some older open pull requests; this check may
+    # need to be adjusted.
+    if int(issue) < 28525:
```

Comment:
Please revert these unrelated changes.

Reply:
This is needed to pass the tests due to the addition of a new NEWS entry that does not satisfy the old assumption.

Reply:
If you pull this out into its own PR we can go ahead and get this tooling bit in as its own change. That may help draw more attention and awareness of the issue to others anyways.

Reply:
I'm not sure there is much benefit in separating this out.

The previous value was just a heuristic anyway --- I guess the author assumed there wouldn't be any older PRs merged afterwards.  I just substituted in the number of this PR to fix the issue here. 

Since all new bugs.python.org ids also are greater than both 32426 and 28525, I'm not sure this check is really serving any purpose, and perhaps should just be removed, but I just wanted to make the minimal change to avoid issues with this PR.

Reply:
edit reverted.  unneeded.  the problem was with the blurb filename which didn't use the issue number.